src/transformState.js

```diff
@@ -6,6 +6,35 @@
  */
 function transformState(state, actions) {
   // write code here
+  const addProperties = 'addProperties';
+  const removeProperties = 'removeProperties';
+  const clear = 'clear';
+
+  for (const action of actions) {
+    switch (true) {
+      case (action.type === addProperties):
```

Comment:
Так теряется вся суть использования switch case. 
```suggestion
    switch (action.type) {
      case (addProperties):
```